tools/src/main/java/com/graphhopper/tools/Measurement.java

```diff
@@ -510,4 +525,75 @@ private void store(FileWriter fileWriter, String comment) throws IOException {
         }
         fileWriter.flush();
     }
+
+    /**
+     * Writes a selection of measurement results to a single line in
+     * a file. Each run of the measurement class will append a new line.
+     */
+    private void writeSummary(String summaryLocation, String propLocation) {
+        // choose properties that should be in summary here
+        String[] properties = {
```

Comment:
Here we can choose the properties that are written to the summary

Reply:
Would it be possible to add a timestamp? I think this could really help when looking at historical data, for example write to the same summary over several weeks/days and easily spot when the run was executed. WDYT?

Reply:
right now we get an 'indirect' timestamp via the .properties file name, but yeah I think adding a proper timestamp is a good idea. will do.